src/styles.scss

```diff
@@ -3,7 +3,7 @@
 @import './scss/sofie-colors';
 @import './scss/sofie-page';
 @import './scss/sofie-forms';
-@import './scss/sofie-hover-scrub';
+@import 'scss/sofie-tooltip-hover';
```

Comment:
I think is good to keep imports from same folder to be with same structure, but not mixed. 
`@import 'scss/sofie-tooltip-hover'; `
or
`@import './scss/sofie-tooltip-hover';`

Reply:
Fixed!